services/web/server/src/simcore_service_webserver/socketio/server.py

```diff
@@ -13,14 +15,40 @@ def get_socket_server(app: web.Application) -> AsyncServer:
     return app[APP_CLIENT_SOCKET_SERVER_KEY]
 
 
+async def _socketio_server_cleanup_ctx(_app: web.Application) -> AsyncIterator[None]:
+    yield
+    # NOTE: this is ugly. It seems though that python-enginio does not
+    # cleanup its background tasks properly.
+    # https://github.com/miguelgrinberg/python-socketio/discussions/1092
```

Comment:
Suggestion: maybe you could also link this PR to the issue, to help others.

Reply:
Yes, I noticed a warning in the logs when a test finishes. There are tasks in the loop added by engineio that are not removed.

Reply:
added link